nncf/quantization/algorithms/weight_compression/algorithm.py

```diff
@@ -403,6 +407,11 @@ def apply(
                 backend_entity=self._backend_entity,
             )
 
+        lora_algorithm = None
+        if self._lora_correction:
+            lora_params = self._advanced_parameters.lora_params
+            lora_algorithm = LoraCorrectionAlgorithm(activations, lora_params)
```

Comment:
@alexsu52 
Introduced `LoraAlgorithm` wrapper just for simplicity. The algorithm can be applied to the layer without it.